### ISSUE TITLE
xiwi cannot start

### ISSUE THREAD

**Issue by roleyzhang** [drinkcat/chroagh#95]
Firstly I installed the arch using command "sudo sh -e installer/main.sh -r arch -t xiwi"
then I installed crouton integration extension on my chromebook, when I finished installation I tried to start xiwi but no starting script in /usr/local/bin/, so I ran 'sudo /usr/local/bin/enter-chroot' then I ran 'xinit'.  crouton integration extension tried to connect to arch but Chrome extension shows this error message.

ERROR: Connection failed (code: -2).
Disconnected, please close the window.

Please help me, Thanks!

**Comment by tvespasian** [comment]
Looks like the most recent Chrome update did this to some folks.  I WAS running very nicely w lxqt, yaourt (to fetch and build it) et al, and had this stable, reliable setup for nigh a year, then SPLAT, the DOWNgrade to v 44.0.2403 fuktover my lovely chroagh bigtime.    First, lxqt wouldn't connect to Xephyr, so I twiddled and tweaked in /etc/crouton to no avail, saw the stuff on this new extension which X clients now are supposed to connect to, wiped my trusty old chroot, got the latest chroagh, noted that yes, it runs boring old XFCE as advertized, but for OTHER window managers or desktops.?.... NADA!!  I'm about 5 attometers away from wiping the eMMC, then putting SLACKWARE on and having DONE with corporate SHIT code that bites folks arses.  Will maybe post some details if I find any cause for hope in this wiki....

**Comment by jasonh9** [comment]
yup my chrome got updated and now my arch awesome wm just went splat.

**Comment by tvespasian** [comment]
No hope yet either.  I got boring old XFCE going after deleting my lovely lxqt env, and updating chroagh/crouton, but this new xiwi extension SHIT doesn't begin to make up for the loss.  I have updated my Slackware sdcard, and it runs OK, though slow, being constrained by the card.  Will likely go ahead and just burn all ChromeOS off the eMMC next week.  Google has shown its hand very clearly now.

**Comment by jasonh9** [comment]
here is a possible fix, I am currently trying this right now. I will update as soon as I see some progress.
https://github.com/dnschneid/crouton/pull/1973

**Comment by jasonh9** [comment]
here is the fix:

(the only option as of now is to downgrade your chromeos version)

instructions:

Downgrading chromeos and then re-installing worked.
Here's a link:
http://www.chromestory.com/2014/11/chromebooks-get-options-downgrade-operating-system/
Also, you can prevent chromeos auto updates as described here:
https://productforums.google.com/forum/#!topic/chromebook-central/KZoIpuuYHeo
(see last comment)

**Comment by tvespasian** [comment]
Hmm, yeah, I DID consider downgrading ChromeOS.  Its a workaround.  Thanks bro.  Happy hacking!  Think I'll go for ArnoldtheBat's ARM build if I flash on any more Chrome(ium) OSes.  More likely, will Slack it.  XFCE works OK w xiwi for this week anyway...
